docs/developer-guide/shipping-bytecode.md

```diff
@@ -36,35 +36,34 @@ a `.o` extension) placed at the root of the layer `./`.
 To provide relevant metadata regarding the bytecode to any consumers, some relevant labels
 **MUST** be defined on the image.
 
-These labels are defined as follows:
+These labels are dynamic and defined as follows:
 
-- `io.ebpf.program_type`: The eBPF program type (i.e `xdp`,`tc`, `sockops`, ...).
+- `io.ebpf.programs`: A label which defines The eBPF programs stored in the bytecode image.
+   the value of the label is a list which must contain a valid JSON object with
+   Key's specifying the program name, and values specifying the program type i.e:
+   "{ "pass" : "xdp" , "counter" : "tc", ...}".
 
-- `io.ebpf.filename`: The Filename of the bytecode stored in the image.
-
-- `io.ebpf.program_name`: The name of the eBPF Program represented in the bytecode.
-
-- `io.ebpf.bpf_function_name`: The name of the function that is the entry point for the BPF program.
+- `io.ebpf.maps`: A label which defines The eBPF maps stored in the bytecode image.
+   the value of the label is a list which must contain a valid JSON object with
+   Key's specifying the map name, and values specifying the map type i.e:
+   "{ "xdp_stats_map" : "per_cpu_array", ...}".
 
 ### Building a Backwards compatible OCI compliant image
 
+Bpfman does not provide wrappers around compilers like clang since many eBPF
+libraries (i.e aya, libbpf, cilium-ebpf) already do so, meaning users are expected
+to pass in the correct ebpf program bytecode for the appropriate platform. However,
+bpfman does provide a few image builder commands to make this whole process easier.
+
 An Example Containerfile can be found at `/packaging/container/deployment/Containerfile.bytecode`
```

Comment:
Is this `bpfman/Containerfile.bytecode` now?

Reply:
It's bpfman/Containerfile.bytecode and bpfman/Containerfile.bytecode.multi.arch.

However in the future it will be no containerfile 🥳